docs/about-filecoin/primer.md

```diff
@@ -0,0 +1,147 @@
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [A Guide to Filecoin Storage Mining](https://filecoin.io/blog/filecoin-guide-to-storage-mining/)
+- [How to mine on the testnet](https://docs.filecoin.io/how-to/install-filecoin/)
+- [Troubleshooting mining issues](https://www.youtube.com/watch?v=D9qdv_3yNbM)
+- [Proof of Replication - Technical Report](https://filecoin.io/proof-of-replication.pdf)
+
+### [Economics of Filecoin](https://www.youtube.com/watch?v=fzGlen-LkKA)
+
+In this Zoom presentation, the Filecoin team discusses the mechanisms and models presented in the recent report _Engineering Filecoin’s Economy_. We will also discuss a number of questions submitted by the community. To have your questions covered, please submit your questions in the linked form below.
+
+Length: 2 hours
+
+Supplemental Materials:
+
+- [Introducing the Filecoin Economy](https://filecoin.io/blog/introducing-the-filecoin-economy/)
+- [Engineering Filecoin’s Economy](https://filecoin.io/2020-engineering-filecoins-economy-en.pdf)
+- [Filecoin’s Cryptoeconomics Constructions](https://filecoin.io/blog/filecoin-cryptoeconomic-constructions/)
+
+## The Filecoin ecosystem
+
+### [Filecoin integrations for web3 infrastructure](https://www.youtube.com/watch?v=Q0oe6i7d1u4)
+
+Filecoin presents many promising business opportunities. This session explores a few key Filecoin integration opportunities for pinning services and other businesses looking to integrate Filecoin.
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [What is an IPFS Pinning Service?](https://medium.com/pinata/what-is-an-ipfs-pinning-service-f6ed4cd7e475#:~:text=An%20IPFS%20pinning%20service%20is,running%20your%20own%20IPFS%20nodes.) (Pinata explainer)
+- [IPFS Docs: Persistence, permanence and pinning](https://docs.ipfs.io/concepts/persistence/)
+- [Filecoin Docs: Filecoin-backed pinning services](https://docs.filecoin.io/build/tools/filecoin-backed-pinning-services/)
+- [Developing on Filecoin](https://www.youtube.com/watch?v=aGCpq0Xf-w8) (video)
+
+### [Using Lotus to connect to the Filecoin network](https://www.youtube.com/watch?v=3dkzLJuv1jc&list=PLXzKMXK2aHh43-aCLLuGnny7nhnsf49LO&index=14&t=0s)
+
+Lotus, the reference implementation of the Filecoin protocol, is your path to connecting to the Filecoin network and interacting with your peers\! In this interactive workshop, we’ll go over the architecture of Lotus, walk through installing Lotus and syncing the blockchain, and examine how you can explore Filecoin’s functionality through the Lotus node.
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [Announcing lotus, our first alternate Filecoin implementation](https://filecoin.io/blog/announcing-lotus/)
+- [Filecoin Docs: lotus and go-ipfs interactions](https://docs.filecoin.io/build/examples/network-inspector/lotus-and-go-ipfs-interactions/)
```

Comment:
This is a section of example app rather than a documentation page on how lotus and go-ipfs interact. The description linked is specific to that use-case, not general. Might be confusing.